swagger.yaml

```diff
@@ -1,8 +1,11 @@
 swagger: "2.0"
 info:
-  description: "An API used to query information about datasets published by the ONS."
+  description: "Used to find information about data published by the ONS.
+  `Datasets` are published in unique `versions`, which are categorized by `edition`.
+  Data in each version is broken down by `dimensions`, and a unique combination
+  of dimension `options` in a version can be used to retrieve `observation` level data."
   version: "1.0.0"
-  title: "Dataset API"
+  title: "Find data"
```

Comment:
I think the title of the spec should be the name of the api, see [here](https://swagger.io/specification/#infoTitle)

Because the swagger website has a continuous banner at the top of the page it covers up the part I wanted to show you, so you will need to scroll up a tad :/

Reply:
The reason for this change is to allow the developer site to have more descriptive labels for users to choose between each API. Alternatively we could consider extending the spec if the current fields dont meet our needs - will continue to research to see what options we have.

Reply:
Having investigated further, supporting additional fields is probably a higher burden than we want to maintain. There's just as many examples of more descriptive titles within their context as titles which are strictly the app name, so we'll continue with descriptive titles for now while we wait for results from user testing.